src/components/VideoPlayer/index.js

```diff
@@ -479,10 +479,12 @@ export default class VideoPlayer extends PureComponent {
     } = this.state
 
     const isScreenOpen = screen !== SCREEN_NONE
+    const isScreenBefore = screen === SCREEN_BEFORE
 
     const containerClasses = cn({
       'bc-player': true,
       'bc-player--screen-open': isScreenOpen,
+      'bc-player--beforescreen': isScreenBefore,
```

Comment:
Thoughts on this being updated to be something like `bc-player--beforescreen-open` to match the above line?